src/index.html

```diff
@@ -82,7 +83,8 @@
   </script> -->
   <!-- Including Ck-edior script-->
   <script src="https://cdn.ckeditor.com/4.6.1/full/ckeditor.js"></script>
-  
+  <script src="https://checkout.razorpay.com/v1/checkout.js"></script>
```

Comment:
Please consider loading this when the user is on the donations page as it's only used there. Having this in the index will slow down the initial load.

Reply:
We tried loading this script in donation page but that is causing issue with razorpay popup. This needs to be loaded with initial load only for razorpay to work properly.